src/Microsoft.PowerShell.Commands.Utility/commands/utility/WebCmdlet/Common/WebRequestPSCmdlet.Common.cs

```diff
@@ -682,6 +693,34 @@ protected override void ProcessRecord()
         /// </summary>
         protected override void StopProcessing() => _cancelToken?.Cancel();
 
+        /// <summary>
+        /// Dispose the CmdLet, cleaning up resources if required. The WebSession must be disposed
+        /// if it is not being used as part of a persistent session.
+        /// </summary> 
```

Comment:
Please make the comment better.

Reply:
I have changed the comment, but not sure if it qualifies as better.